edge-modules/api-proxy-module/src/monitors/config_parser.rs

```diff
@@ -9,19 +9,35 @@ use crate::token_service::token_server;
 const PROXY_CONFIG_DEFAULT_VALUES: &[(&str, &str)] = &[("NGINX_DEFAULT_PORT", "443")];
 
 pub struct ConfigParser {
-    context: std::collections::HashMap<String, String>,
+    default_values: HashMap<String, String>,
 }
 
 impl ConfigParser {
-    pub fn new(config: &str) -> Result<Self, anyhow::Error> {
-        let mut context = std::collections::HashMap::new();
-        let mut default_values = std::collections::HashMap::new();
+    pub fn new() -> Result<Self, anyhow::Error> {
+        let mut default_values = HashMap::new();
 
         //Set default config values
         for (key, value) in PROXY_CONFIG_DEFAULT_VALUES.iter() {
             default_values.insert((*key).to_string(), (*value).to_string());
         }
 
+        //Harded coded keys
+        // Tokens are cached by nginx. Current duration is 30mn. Make sure the the validity is not lower than the caching time.
+        default_values.insert(
+            "TOKEN_VALIDITY_MINUTES".to_string(),
```

Comment:
should you move it to PROXY_CONFIG_DEFAULT_VALUES instead?

Reply:
Excellent point will do